tools/generator/src/Generator.swift

```diff
@@ -129,7 +131,10 @@ Was unable to merge "\(srcTarget.label) \
             pbxTargets
         )
 
-         let xcodeProj = environment.createXcodeProj(pbxProj)
+        let schemes = environment.createXCSchemes(disambiguatedTargets)
+        let sharedData = environment.createXCSharedData(schemes)
+
+         let xcodeProj = environment.createXcodeProj(pbxProj, sharedData)
```

Comment:
Can you fix the indenting on this one and below? Not sure how it got the extra space